### DISCUSSION TITLE
Student needing help from software engineers with our project

### DISCUSSION THREAD

**Discussion by jkmorgan22** [community/community#36060] [Other Feature Feedback, Questions, & Ideas]
Hello all 👋, I'm new to GitHub, but I need your help with a research project I'm working on.

I'm in a product development program at my university. My team and I have to build a validated solution that solves a pain for our class project this year. We are building a platform where professional software engineers can control what offers they hear about from recruiters, rather than trying to filter through the spam. 

On our platform, software engineers can show not only their professional experience and top skills, but also a "**Make Me Quit**" price they build themselves. Engineers can set salary minimums, remote/hybrid/in-person expectation, technologies they want to work with, and more to create their Make Me Quit price. Our technology will only allow recruiters to unlock their anonymous profile and contact them 1) if the recruiter's job opening meets the engineer's Make Me Quit criteria and 2) the engineer agrees. 

With our platform, professional engineers **will only get contacted with offers that they might actually consider** leaving their job for. 

We have multiple recruiters who work at some pretty amazing tech companies that are interested in testing our platform. Now, we are looking to get a good number of software engineers (ideally with 3+ years of professional experience) to create beta profiles on our platform so we can validate our idea.

In order to continue our project, we have to show progress and validation. My classmates and I already have a few software engineers who have created beta profiles, but we need more to get our project approved. Our first beta test is just answering a couple questions on this [Google Form](https://forms.gle/uP7WQhWVfiWUk2858) to build a Make Me Quit price and connecting with us on [LinkedIn](https://www.linkedin.com/in/jackmorganstl/). 

If you are a software engineer that's willing to help us out and create a beta profile, **please connect with me** [here](https://www.linkedin.com/in/jackmorganstl/) on my LinkedIn, send me a message, and **fill out this** [Google Form](https://forms.gle/uP7WQhWVfiWUk2858). I would appreciate you sharing this opportunity with any software engineers you know (whether they are looking for a new job or not). 

Software engineers **will not get any spam**, **won't have to pay anything** (we might even give them money for taking interviews), and **may even get a job offer** that is exactly what they are looking for. 

I would appreciate any help that we can get. Please respond with any questions and share this with your software engineer friends 😁 Thank you for helping us with our project!

**Comment by athibau2** [comment]
This is a cool idea, I hope you get some success with it. I know a few people I could send this to, so hopefully that helps you out. Good luck!
